common/Resources/ti.internal/extensions/ti/ti.blob.js

```diff
@@ -4,22 +4,8 @@
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
-/* globals OS_IOS, OS_VERSION_MAJOR */
 const buffer = Ti.createBuffer({ value: '' });
 const blob = buffer.toBlob();
-
-if (OS_IOS) {
```

Comment:
I think it should be here. There were 2 issues - https://jira.appcelerator.org/browse/TIMOB-27707 and https://jira.appcelerator.org/browse/TIMOB-27350 . Though unit test is passing. I'll check for lower iOS versions tomorrow.

Reply:
`toString()` is now implemented natively. As for the hack, I think it just works now?

Reply:
I tested it. It looks for iOS 13+, it is working fine. For iOS < 13, it is not working. We need hack for iOS < 13.